dist/README.md

```diff
@@ -0,0 +1,394 @@
+Authors Haven - A Social platform for the creative at heart.
```

Comment:
Hi. I am not sure your dist/ folder should have been uploaded, can you double-check that and your gitignore file. Thanks

Reply:
I think it is alright to upload your .gitignore file

Reply:
Yes, apologies. I meant double check that your .gitignore file currently includes your `dist/` folder among folders to be ignored.

Reply:
@olajide1234  and @kleva-j. Thanks for the feedback. The dist folder is ignored in my .gitignore file. However, I didn't edit any of those files but got it into my branch when I pulled from develop because I needed to document my API.